services_app/src/main/res/values/strings.xml

```diff
@@ -258,7 +258,7 @@
     <string name="admin_password_disabled_click_to_set">No Admin password. Click to enable.</string>
     <string name="admin_password_settings">Admin Password</string>
     <string name="admin_password">Admin Password</string>
-    <string name="change_admin_password">Change admin password</string>
+    <string name="change_admin_password">Privacy</string>
```

Comment:
Admin password is not privacy ... seems like a change we should discuss at meeting

Reply:
applied label update to wrong string